karma.conf.js

```diff
@@ -18,7 +18,8 @@ module.exports = function(config) {
         // list of files / patterns to load in the browser
         files: [
             'tests_index.js',
-            {pattern: 'test/fixtures/*.html', watched: true, served: true, included: false}
+            {pattern: 'test/fixtures/*.html', watched: true, served: true, included: false},
```

Comment:
Shim needs to be included in `PhantomJS` to accept the `Array.prototype.find` method used in `getNearestDataPoint` function here https://github.com/eventbrite/britecharts/blob/master/src/charts/grouped-bar.js#L607

Reply:
Good catch!